manifests/00-cluster-role.yaml

```diff
@@ -239,6 +239,7 @@ rules:
   - use
   resourceNames:
   - hostnetwork
+  - restricted
```

Comment:
I'm wondering why the router pod is not applicable for to the new `restricted-v2` in case the hostnetwork is not used. And as continuation, if the scc has to be specified explicitly, why not specify the new one instead?

Reply:
Sorry, I see now that Candace had [a similar question](https://github.com/openshift/cluster-ingress-operator/pull/1064#issuecomment-2138149989).

> Crucially, the "restricted-v2" SCC prohibits enabling allowPrivilegeEscalation, which router pods require.

Right.